packages/xrpl/src/models/ledger/Ledger.ts

```diff
@@ -61,5 +62,7 @@ export default interface Ledger {
    * either JSON or binary depending on whether the request specified binary
    * as true.
    */
-  transactions?: Array<Transaction & { metaData?: TransactionMetadata }>
+  transactions?: Array<
+    (Transaction | PseudoTransaction) & { metaData?: TransactionMetadata }
```

Comment:
```suggestion
    (Transaction | PseudoTransaction) & { metaData?: TransactionMetadata }
```
Is it possible to combine `Transaction | PseudoTransaction` into one type? I see typing also occurs `hashTxTree`